descriptors/ModuleDescriptor-template.json

```diff
@@ -14,7 +14,7 @@
   "provides": [
     {
       "id": "notes",
-      "version": "2.0",
+      "version": "3.0",
```

Comment:
This version change should be reflected in the YAML file describing the interface

(it is currently version 1.0, which suggests to me that the previous breaking change also did not change this)